language_common.md

```diff
@@ -0,0 +1,6 @@
+# Standards common to coding in any language
+
+## Avoid using 'magic' numbers
+Having numbers in code without an indication of what they mean can be unhelpful when reading the code when a change is needed. The preferred solution is to create constants with descriptive names for the numbers.
```

Comment:
Pehaps:
... is to create constants with descriptive names for the numbers. If the numbers only have a limited set of allowed values, you should consider using an enum instead.

Reply:
Are enums Java specific? This README is for common things.

Reply:
Hmmm. How about:
... is to create constants with descriptive names for the numbers. If the numbers only have a limited set of allowed values, you should consider using an enum instead (if your language allows).